.github/workflows/pr-check.yaml

```diff
@@ -146,46 +159,93 @@ jobs:
           run_install: false
           package_json_file: ./podman-desktop/package.json
 
-      - name: Execute pnpm
+      - name: Install Podman Desktop dependencies
         working-directory: ./podman-desktop
         run: pnpm install
 
+      - name: Install SSO extension dependencies
+        if: matrix.os == 'windows-2022'
+        working-directory: ./sso-extension
+        run: pnpm install
+
       - name: Build Podman Desktop for E2E tests
         working-directory: ./podman-desktop
         run: pnpm test:e2e:build
+      
+      - name: Adjust/Downgrade local podman desktop version Windows
```

Comment:
Why this change?

Reply:
That's what you suggested me I should use to fix the encoding issues I had on the package.json from windows

Reply:
It was that you use the command, how it uses jq tool, not that it should downgrade the podman-desktop, or in this case, you are downgrading crc-extension.